content/docs/user-guide/related-technologies.md

```diff
@@ -0,0 +1,133 @@
+# Comparison with Related Technologies
+
+DVC combines a number of existing ideas into a single tool, with the goal of
+bringing best practices from software engineering into the data science field
+(refer to [What is DVC?](/doc/user-guide/what-is-dvc) for more details).
+
+## Git
+
+- DVC builds upon Git by introducing the concept of data files – large files
+  that should not be stored in a Git repository, but still need to be tracked
+  and versioned. It leverages Git's features to enable managing different
+  versions of data itself, data pipelines, and experiments.
+
+- DVC is not fundamentally bound to Git, and can work without it (except
+  versioning-related features). This also applies to Git-LFS and Git-annex,
+  below.
+
+## Git-LFS (Large File Storage)
+
+- DVC does not require special servers like Git-LFS demands. Any cloud storage
+  like S3, Google Cloud Storage, or even an SSH server can be used as a
+  [remote storage](/doc/command-reference/remote). No additional databases,
+  servers, or infrastructure are required.
+
+- DVC does not add any hooks to the Git repo by default (although they are
+  [available](/doc/command-reference/install)).
+
+- Git-LFS was not made with data science in mind, so it doesn't provide related
+  features (e.g. [pipelines](/doc/command-reference/dag),
+  [metrics](/doc/command-reference/metrics), etc.).
+
+- Github (most common Git hosting service) has a limit of 2 GB per repository.
```

Comment:
minor: GitHub

Reply:
Yeah I think both are OK. Technically Github is more correct writing-wise. We have a mix of both forms rn though, I'll standardize in another PR. Will use GitHub since you prefer that 🙂